webapps-schema/src/main/java/io/camunda/webapps/schema/entities/usermanagement/GroupEntity.java

```diff
@@ -34,12 +35,25 @@ public GroupEntity setName(final String name) {
     return this;
   }
 
-  public Set<Long> getAssignedMemberKeys() {
-    return assignedMemberKeys;
+  public Long getEntityKey() {
+    return entityKey;
+  }
+
+  public GroupEntity setEntityKey(final Long entityKey) {
```

Comment:
I think this property should be getMemberKey/setMemberKey for consistency with the field in ES. We could rename the existing getMemberKey to getChildKey.